languages/elixir/exercises/concept/lists/.docs/after.md

```diff
@@ -0,0 +1,33 @@
+## Key Points
```

Comment:
I really like the `after.md` document! Are there documents you could refer the student to at this point? E.g. official documentation like https://hexdocs.pm/elixir/List.html.